docs/ManagingReleases.md

```diff
@@ -0,0 +1,66 @@
+# Managing Releases
+
+* **Who does it?** Releases can be managed by the _Release Lead_ of the project (if there is one) or the project lead.
+
+* **How often?** Weekly time-boxed release cycles are encouraged. Projects that have no end-user facing 
+  product may not have releases.
+
+* **What to include in a release?** Planning a release in advance is not practical if code contributions
+  are intermittent and the available manpower for a given release cycle is not always predictable. 
+  In those cases the release can contains whatever new code that has been added since the previous release. 
+  However, major releases which are not backward compatible may need planning in advance.
+
+* **Maintaining project velocity**: 
+  It is important to keep resolving issues at regular intervals if the project were to keep moving 
+  forward at a reasonable pace. Recommendations: 
+  * Shorten the lifecycle of ongoing PRs by aggressively reminding dev/reviewers to resume work when the PR progress 
+    becomes stagnant. 
+    * Project lead needs to periodically check the issue tracker for inactive PRs and remind the relevant 
+      persons to resume work.
+    * If a PR is inactive even after a reminder, it can be closed (and the issue unassigned). 
+    * TODO: automate the reminders.
```

Comment:
This seems like it is for our projects, so it should be removed.
Otherwise, it should be made generic like "Some third-party services or bots may be able to automate those reminders".